spring-cloud/spring-cloud-stream/spring-cloud-stream-rabbit/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>spring-cloud-stream-rabbit</artifactId>
+    <packaging>jar</packaging>
+
+    <name>spring-cloud-stream-rabbit</name>
+    <description>Simple Spring Cloud Stream</description>
+
+    <parent>
+        <groupId>org.baeldung</groupId>
+        <artifactId>spring-cloud-stream</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>..</relativePath>
+    </parent>
+
+    <dependencies>
+   
+        <dependency>
```

Comment:
use only spaces for indentation -- not tabs

Reply:
Ok

Reply:
Ok